### ISSUE TITLE
[Feature Request]: Add check for whitespace after type casting

### ISSUE THREAD

**Issue by tyrann0us** [inpsyde/php-coding-standards#55]
### Is your feature request related to a problem?

Currently, there is no rule that checks whether there is a space after the type casting. As a result, several spellings exist.

Without a space:
```php
return (int)$lowStockThreshold;
```

With spaces:
```php
return (string) max($lowStockThresholds);
```

### Describe the desired solution

Add a rule that checks that there is a space after the type casting.

### Describe the alternatives that you have considered

None.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by gmazzap** [comment]
Because both spellings exists, we can't decide which one is to prefer. We need to gather consensus.

**Comment by luislard** [comment]
Cross referencing this:

https://www.php-fig.org/per/coding-style/#:~:text=Type%20casting%20operators%20MUST%20NOT%20have%20any%20space%20within%20the%20parentheses%20and%20MUST%20be%20separated%20from%20the%20variable%20they%20are%20operating%20on%20by%20exactly%20one%20space

**Comment by gmazzap** [comment]
Fixed in #76
